integrations/fake-integration/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,30 @@
+resources:
+  - kind: fake-person
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"fake-person"'
+          properties:
+            name: .name
+            email: .email
+            status: .status
+            age: .age
+            department: .department.name
+        relations:
+          department: .department.id
+  - kind: fake-department
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"fake-department"'
+          properties:
+            name: .name
+            id: .id
```

Comment:
order should be the opposite? as that the order in which we sync is ordered by the list, so you are trying to map fake user to department that wasn't created yet